docs/topics/appendix-proficiency-levels.adoc

```diff
@@ -0,0 +1,21 @@
+
+[[proficiency_levels]]
+= Proficiency Levels
+
+Using each mission available on {launcher} requires certain minimum knowledge from you, and provides a certain information to you. For easier understanding, the requirements and concepts are organized in several levels. In addition to the levels described in this chapter, there can be additional requirements with each mission, specific to its aim or the technologies it uses.
```

Comment:
@rhoads-zach Revised. Note that we should not use "may" except when it literally means "it is allowed to" [IBM Style Guide].